chain-api/src/types/mint.ts

```diff
@@ -484,3 +484,17 @@ export class FetchTokenMintConfigurationsResponse extends ChainCallDTO {
   @IsString()
   bookmark: string;
 }
+
+export class DeleteTokenMintConfigurationDto extends SubmitCallDTO {
```

Comment:
It is required to provide some additional dtos, since the default used (like `TokenClassKey` does not have `uniqueKey` defined)